pgactivity/queries/get_blocking.sql

```diff
@@ -17,17 +17,19 @@ SELECT
       END AS state,
       CASE WHEN sq.query LIKE '<IDLE>%%'
           THEN NULL
-          ELSE sq.query
+          ELSE convert_from(sq.query::bytea, coalesce(pg_catalog.pg_encoding_to_char(b.encoding), 'UTF8'))
       END AS query,
       waiting AS wait
   FROM
       (
+      -- Transaction id lock
       SELECT
             blocking.pid,
             '<unknown>' AS application_name,
             pg_stat_activity.current_query AS query,
             blocking.mode,
             pg_stat_activity.datname,
+	    pg_stat_activity.datid,
```

Comment:
There appears to be a leading tab here (and everywhere else below); can you use spaces for consistency?

Reply:
Fixed this on + some in get_blocking_post_90200.sql and get_pg_activity.sql in  d1140fc